test/gpu/gemm_tune.cpp

```diff
@@ -130,7 +131,7 @@ TEST_CASE(gemm_tune_strided)
             break;
         }
     }
-#ifdef MIGRAPHX_USE_ROCBLAS_TUNING_API
+#if defined(MIGRAPHX_USE_ROCBLAS_TUNING_API) or defined(MIGRAPHX_USE_HIPBLASLT)
```

Comment:
After using `target_compile_definitions` to set `MIGRAPHX_USE_HIPBLASLT` in the makefile, it appears to me that it will always be defined and set to either 0 or 1 at this point, which would make this `#if` always true.  Can you confirm that the code really doesn't get compiled when we don't want it to?

Reply:
Thanks for catching this. I will update this.

Reply:
I have updated it to check for the value instead of definition.

Reply:
I wish we had a convention for preprocessor definitions that would make this mistake less likely, but the fix looks good.